.gitignore

```diff
@@ -37,4 +37,7 @@ docker/volumes/**/*
 # Ignore any hidden files/folders (starting with a .)
 **/.*
 # except the github dir...
-!.github 
\ No newline at end of file
+!.github 
+
+# Ignore any docker env files
+*.env
```

Comment:
Thinking about it, we won't need this as we're already ignoring files starting with a dot, above